docs/source/tutorials/e2e_flow.rst

```diff
@@ -25,222 +25,239 @@ from the community seemlessly with torchtune.
-
-    [evaluator.py:324] Running loglikelihood requests
-    [eleuther_eval.py:195] Eval completed in 121.27 seconds.
-    [eleuther_eval.py:197] truthfulqa_mc2: {'acc,none': 0.388...
+Then, in your config, you only need to replace two fields: ``output_dir`` and ``checkpoint_files``. Notice
+that we are using the merged weights, and not the LoRA adapters.
 
-The model has an accuracy around 38.8%. Let's compare this with the fine-tuned model.
+.. code-block:: yaml
 
+    # TODO: update to your desired epoch
+    output_dir: /tmp/torchtune/llama3_2_3B/lora_single_device/epoch_0
 
-First, we modify ``custom_eval_config.yaml`` to include the fine-tuned checkpoints.
+    # Tokenizer
+    tokenizer:
+        _component_: torchtune.models.llama3.llama3_tokenizer
+        path: ${output_dir}/original/tokenizer.model
 
-.. code-block:: yaml
+    model:
+        # Notice that we don't pass the lora model. We are using the merged weights,
+        _component_: torchtune.models.llama3_2.llama3_2_3b
 
     checkpointer:
         _component_: torchtune.training.FullModelHFCheckpointer
-
-        # directory with the checkpoint files
-        # this should match the output_dir specified during
-        # finetuning
-        checkpoint_dir: <checkpoint_dir>
-
-        # checkpoint files for the fine-tuned model. This should
-        # match what's shown in the logs above
+        checkpoint_dir: ${output_dir}
         checkpoint_files: [
-            hf_model_0001_0.pt,
-            hf_model_0002_0.pt,
+            ft-model-00001-of-00002.safetensors,
+            ft-model-00002-of-00002.safetensors,
         ]
+        output_dir: ${output_dir}
+        model_type: LLAMA3_2
 
-        output_dir: <checkpoint_dir>
-        model_type: LLAMA2
+    ### OTHER PARAMETERS -- NOT RELATED TO THIS CHECKPOINT
 
-    # Make sure to update the tokenizer path to the right
-    # checkpoint directory as well
-    tokenizer:
-        _component_: torchtune.models.llama2.llama2_tokenizer
-        path: <checkpoint_dir>/tokenizer.model
+    # Environment
+    device: cuda
+    dtype: bf16
+    seed: 1234 # It is not recommended to change this seed, b/c it matches EleutherAI's default seed
 
+    # EleutherAI specific eval args
+    tasks: ["truthfulqa_mc2"]
+    limit: null
+    max_seq_length: 4096
+    batch_size: 8
+    enable_kv_cache: True
 
-Now, let's run the recipe.
-
-.. code-block:: bash
-
-    tune run eleuther_eval --config ./custom_eval_config.yaml
+    # Quantization specific args
+    quantizer: null
 
+For this tutorial we'll use the `truthfulqa_mc2 <https://github.com/sylinrl/TruthfulQA>`_ task from the harness.
 
-The results should look something like this.
+This task measures a model's propensity to be truthful when answering questions and
+measures the model's zero-shot accuracy on a question followed by one or more true
+responses and one or more false responses
 
-.. code-block:: bash
+.. code-block:: text
```

Comment:
Maybe ignore my above comments, does bash render weirdly or something?

Reply:
Yeah when you display the outputs from bash.